.github/CODEOWNERS

```diff
@@ -1,27 +1,40 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-/global.json                    @aspnet/build
```

Comment:
@Pilchie same question here as in #35950 about you owning the world

Reply:
> @Pilchie same question here as in #35950 about you owning the world

Fine with me.